File: ios/HelloWorld/HelloWorld/HelloWorldScreen.swift
//Copyright (c) 2020 Everysight LTD

import Foundation
import NativeEvsKit

class HelloWorldScreen : Screen {
    
    let text = Text()
    override func onCreate() {
        text.setText(text: "Hello World")
            .setResource(font: Font(stockFont: .medium))
            .setTextAlign(align: Align.center)
            .setForegroundColor(color: EvsColor.green.rgba)
        text.setX(x: getWidth()/2)
        text.setY(y: getHeight()/2)
        add(uiElement: text)
    }
    
}

File: android/helloworld/src/main/java/com/everysight/sdk/samples/helloworld/HelloWorldScreen.kt
package com.everysight.sdk.samples.helloworld

import UIKit.app.Screen
import UIKit.app.data.Align
import UIKit.app.data.EvsColor
import UIKit.app.resources.Font
import UIKit.widgets.Text


class HelloWorldScreen:Screen() {
    private val text = Text()
    override fun onCreate() {

        text.setText("Hello World").setResource(Font.StockFont.Medium).setTextAlign(Align.center)
        text.setX(getWidth()/2).setY(getHeight()/2).setForegroundColor(EvsColor.Green.rgba)
        add(text)
    }
}